lib/haskell/natural4/test/LS/XPile/GenericMathLang/TranslateL4Spec.hs

```diff
@@ -325,12 +327,16 @@ spec = do
               , ("accident.juvenile limit applies" , PredVal Nothing True)
               , ("accident.triple benefits apply" , PredVal Nothing True)
               , ("accident.double benefits apply" , PredVal Nothing False)
-              , ("illness.disqualified" , PredVal Nothing False)
+              -- , ("illness.disqualified" , PredVal Nothing False)
```

Comment:
The reason I did this was because I originally tried to replace the PAU case with a textual grammar, but noticed that the whole thing is buggy in ways unrelated to the BNFC grammar. So I decided to leave the code as is, and instead wrote a test that only turns TextuaL4 into Rule AST, and that test is in TextuaL4Spec.hs.